README.md

```diff
@@ -296,6 +298,71 @@ and then change its colors, the items missed would fallback to use the default c
 Please also notice that an empty theme is **NOT** supported due to
 [a bug in serde lib](https://github.com/dtolnay/serde-yaml/issues/86).
 
+### Icon Theme
+
+Icon theme can be configured in a fixed location, `LSD_CONFIG_DIR/themes/icons.yaml`,
```

Comment:
Why not `LSD_CONFIG_DIR/icons.yaml`, do we need a subdirectory. The way I see it, we will have 3 top level files:
- `config.yaml`
- `colors.yaml`
- `icons.yaml`

> We will have to make changes to the color one as well, but yeah.